site/docs/v1.3.0/httpproxy.md

```diff
@@ -1263,6 +1263,35 @@ spec:
             subjectName: foo.marketing
 ```
 
+## Validation of External Client Certificate
```

Comment:
Lets title this "Client Certificate Validation"?

Reply:
done!